src/config/config.ts

```diff
@@ -402,26 +242,25 @@ const otpGenerator = () => {
 }
```

Comment:
probably need to fix merge conflicts here, as `otpGenerator` has been moved from this file

Reply:
good catch. will do